lib/genevalidator/ext/array.rb

```diff
@@ -21,7 +21,7 @@ def mode
     end
 
     def sample_variance
-      m   = mean
+      m   = inject(:+).to_f / size
```

Comment:
I tried this, but I couldn’t get it to work - https://github.com/wurmlab/genevalidator/tree/mean